test/util/Fixtures.scala

```diff
@@ -102,6 +102,8 @@ trait TaiFixtures {
 
 trait TaxCalculationFixtures {
   def buildTaxCalculation = TaxCalculation("Overpaid", BigDecimal(84.23), 2015, Some("REFUND"), None, None, None)
+  def buildTaxYearReconciliations: List[TaxYearReconciliation] =
+    List(TaxYearReconciliation(2015, Balanced), TaxYearReconciliation(2016, Balanced))
 }
```

Comment:
This is only used in one place, does it need to be in fixtures?

Reply:
This is now used in multiple places